### PR TITLE
Add vaccines show page

### PR DESCRIPTION
**Pull request by misaka**
Adds show page for vaccines.

image

The layout is currently two-thirds but looks like it should be three-quarters looking at the prototype. To be addressed later. The prototype also adds the "Dose" field and pre-screening questions, which we don't have yet. Will add these later.

### CODE REVIEW

File: app/controllers/vaccines_controller.rb [hunk 1]
```diff
@@ -1,8 +1,14 @@
 class VaccinesController < ApplicationController
   include TodaysBatchConcern
 
+  layout "two_thirds"
```
Comment (misaka) [inline app/controllers/vaccines_controller.rb:4]: This doesn't look quite right , but it's broken in the same way other pages are broken ... I think the prototype uses a three-quarters layout? 🥾 🥫

Comment (tvararu) [reply app/controllers/vaccines_controller.rb:4]: Kick the can yeah, we don't have the 3-quarters layout yet, asked on Slack about it

File: spec/features/manage_vaccines_spec.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  def then_i_see_an_hpv_vaccine_listed
+    expect(page).to have_content("Gardasil 9 (HPV)")
+  end
+
+  def when_i_click_on_the_vaccine
+    click_on "Gardasil 9 (HPV)"
+  end
+
+  def then_i_see_the_vaccine_details
+    expect(page).to have_selector :heading, "Gardasil 9 (HPV)"
```
Comment (misaka) [inline spec/features/manage_vaccines_spec.rb:38]: @tvararu This is slightly new, dunno how you feel about testing this way but let me know.

File: app/helpers/vaccines_helper.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+module VaccinesHelper
+  def vaccine_heading(vaccine)
+    sprintf("%s (%s)", vaccine.brand, t(vaccine.type, scope: "vaccines"))
```
Comment (tvararu) [inline app/helpers/vaccines_helper.rb:3]: Yeah, `sprintf` is clearer than string interpolation isn't it 👍

Comment (misaka) [reply app/helpers/vaccines_helper.rb:3]: In this case. I wanted to do `"%s (%s)" % [vaccine.brand, t(vaccine.type, scope: "vaccines")]" but Rubocop got pushy about it ... I think it has a chip on it's shoulder about looking too much like Python.

**Review by tvararu — APPROVED** [review]

File: spec/features/manage_vaccines_spec.rb (hunk 2 above)
Comment (tvararu) [reply spec/features/manage_vaccines_spec.rb:38]: LGTM

**Pull request merged**